Demo/UIKit/CustomComponent/CustomComponentPresenter.swift

```diff
@@ -5,25 +5,63 @@
 //
 
 import Adyen
+import AdyenActions
 import AdyenNetworking
-
-protocol CustomComponentPresenterProtocol {}
+import AdyenSession
+@testable @_spi(AdyenInternal) import Adyen
+@testable import AdyenCard
+
+protocol CustomComponentPresenterProtocol {
+    var cardViewController: UIViewController { get }
+    func performPayment()
```

Comment:
`performPayment` doesn't sound as a presenter responsibility, does it? If this triggers payment floe presentation, maybe let's try something like `showPaymentFlow()`?

Reply:
In this case, `performPayment` triggers the payment flow (payment request). There's no UI involved.

Reply:
I think I am still thinking in viper presenters, sorry. I see we use more MVP approach here.